rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_reader :book, :person
+
+  def initialize(book, person)
```

Comment:
- [ ] The @date instance variable is initialized in the constructor since it will be need to create a rental. Something like so: 
`  def initialize(date)`
   ` @date = date`
 ` end`

Reply:
Thank you so much for this review, I have implemented successfully